README.md

```diff
@@ -22,6 +22,14 @@ If the pipeline is backed by Orchestra, the latest published version will run. I
 
 (Optional) The environment name or ID to run the pipeline in. If not specified, the default environment will be used
 
+### `task_ids`
+
+(Optional) If the user wants to run a subset of tasks in the pipeline, they can specify the IDs of the tasks to run. No task group ids are supported.
```

Comment:
given this is public docs, let's not refer to "the user" and instead use more active wording. also i assume it has to be comma separated?

i.e. `To run a subset of tasks of tasks in the pipeline, specify the task IDs separated by a comma. No task group ids are supported.`